modules/apps/27-interchain-accounts/controller/keeper/msg_server_test.go

```diff
@@ -25,46 +27,46 @@ func (suite *KeeperTestSuite) TestRegisterInterchainAccount_MsgServer() {
 
 	testCases := []struct {
 		name     string
-		expPass  bool
+		expErr   error
 		malleate func()
 	}{
 		{
 			"success",
-			true,
+			nil,
 			func() {},
 		},
 		{
 			"success: ordering falls back to UNORDERED if not specified",
-			true,
+			nil,
 			func() {
 				msg.Ordering = channeltypes.NONE
 				expectedOrderding = channeltypes.UNORDERED
 			},
 		},
 		{
 			"invalid connection id",
-			false,
+			connectiontypes.ErrConnectionNotFound,
 			func() {
 				msg.ConnectionId = "connection-100"
 			},
 		},
 		{
 			"non-empty owner address is valid",
```

Comment:
Maybe we could also move this test up together with the rest of success cases.

Reply:
Yeah I actually wanted to do this but also wanted to keep the diff tidy! Now that it has been reviewed I think it makes more sense to move it :)